Resources/public/js/views/ez-languageselectionboxview.js

```diff
@@ -0,0 +1,418 @@
+                this.get('container')
+                    .one('.ez-' + section + '-translation[data-languagecode="' + languageCode + '"]')
+                    .addClass(TRANSLATION_HIGHLIGHT_CLASS);
+            }
+        },
+
+        /**
+         * Sets the state for base translation checkbox
+         *
+         * @method _uiBaseTranslationState
+         * @protected
+         * @param {String} baseTranslation
+         */
+        _uiBaseTranslationState: function (baseTranslation) {
+            var c = this.get('container'),
+                baseCheckbox = c.one('.ez-base-translation-checkbox');
+
+            baseCheckbox.set('checked', baseTranslation);
+            if (baseTranslation) {
+                c.addClass(BASE_TRANSLATIONS_LIST_VISIBLE_CLASS);
+            } else {
+                c.removeClass(BASE_TRANSLATIONS_LIST_VISIBLE_CLASS);
+            }
+        },
+
+        /**
+         * Sets visibility of whole part containing base translations
+         *
+         * @method _uiSetBaseTranslationVisibility
+         * @protected
+         * @param {Boolean} canBaseTranslation
+         */
+        _uiSetBaseTranslationVisibility: function (canBaseTranslation) {
+            var c = this.get('container');
+
+            if (canBaseTranslation) {
+                c.addClass(BASE_TRANSLATION_AVAILABLE_CLASS);
+            } else {
+                c.removeClass(BASE_TRANSLATION_AVAILABLE_CLASS);
+            }
+        },
+
+        /**
+         * `selectedNewTranslationChange` event handler. It enables/disables the button
+         * depending on new translationselection
+         *
+         * @method _uiSetConfirmButtonState
+         * @protected
+         */
+        _uiSetConfirmButtonState: function () {
+            var confirmButton = this.get('container').one('.ez-languageselectionbox-confirm');
+
+            confirmButton.set('disabled', !this.get('selectedNewTranslation'));
+        },
+
+        render: function () {
+            var container = this.get('container');
+
+            container.setHTML(this.template({
+                title: this.get('title'),
+                newTranslations: this.get('newTranslations'),
+                existingTranslations: this.get('existingTranslations'),
+                canBaseTranslation: this.get('canBaseTranslation'),
+            }));
+
+            return this;
+        },
+
+        /**
+         * Cancel the languageSelectionBox by firing the `cancelLanguageSelect` event
+         *
+         * @protected
+         * @method _cancelLanguageSelection
+         */
+        _cancelLanguageSelection: function () {
+            this.fire('cancelLanguageSelect');
+        },
+
+        /**
+         * Confirm the languageSelectionBox's selection by firing the `languageSelected` event
+         *
+         * @protected
+         * @method _confirmLanguageSelection
+         */
+        _confirmLanguageSelection: function () {
+            this.fire('languageSelected');
```

Comment:
Is it really needed? Currently only thing that `languageSelected` event is doing is hiding sideView without using any of attributes. What's more this function `_confirmLanguageSelection` is only default function and it should be overriden by passing own function to `confirmHandler`.